source/core.py

```diff
@@ -416,9 +416,9 @@ def handlePowerStatusChange(self):
 		log.warning("Java Access Bridge not available")
 	except:
 		log.error("Error initializing Java Access Bridge support", exc_info=True)
-	import winConsoleHandler
-	log.debug("Initializing winConsole support")
-	winConsoleHandler.initialize()
+	import winConsoleHandlerLegacy

```

Comment:
Why did you rename this as well?

Reply:
To avoid confusion (in anticipation of UIA becoming the default).